src/resources/assets/js/mixins/largoContainer.vue

```diff
@@ -139,7 +138,7 @@ export default {
             return annotations;
         },
         hasNoAnnotations() {
-            return this.selectedLabel && !this.loading && this.annotations.length === 0;
+            return this.selectedLabel && !this.loading && this.sortedAnnotations.length === 0;
```

Comment:
Why this change? Shouldn't the annotations and sorted annotations have equal length?